frontend/src/helpers/Analytics.ts

```diff
@@ -147,9 +149,27 @@ export class Analytics {
     window.analytics.track(name, context)
   }
 
-  public trackConnect(name: string, data?: { id: string; name: string; typeID?: number }, error?: ISimpleError) {
-    if (data) this.track(name, { ...data, error, connectionType: CONNECTION_TYPE_FAILOVER })
+  public trackConnect(name: string, data?: { id: string; name: string; typeID?: number, isP2P?: boolean }, error?: ISimpleError) {
+    if (data) {
+      const connectionType = data?.isP2P ? CONNECTION_TYPE_PEER : CONNECTION_TYPE_FAILOVER
+      this.track(name, { ...data, error, connectionType })
+    }
+  }
+
+  public trackConnect(
```

Comment:
@mauricio0000 Maybe I'm missing something... but looks like there are two methods with the same name: `trackConnect` that' can't be right?

Reply:
@JamieRuderman it was produced by the rebase, I remove the duplicate code